setup.py

```diff
@@ -2,13 +2,13 @@
 
 setup(
     name='synology-api',
-    version='0.0.3',
+    version='0.1.0',
     packages=find_packages(exclude=['tests*']),
     license='MIT',
     description='Python Synology API Wrapper',
     long_description=open('README.md').read(),
     install_requires=['requests'],
-    url='https://github.com/N4S4/synology-api',
-    author='Renato Visaggio',
-    author_email='renatovisaggio@gmail.com'
+    url='https://github.com/emelbaer/synology-api',
+    author='Emel B (c) Renato Visaggio',
```

Comment:
The following are just little requests as I spent a considerable amount of time reading the docs and writing this wrapper even if is not perfect at all, 

I will soon write some contributing guide to the package, for now I will write just a comment on how I would like it to be done if is requested to merge to the master branch;


- I would keep major version changes for when the api wrapper is complete in all its functions. 
- url has to be referred to the master branch

- Author email has to stay, if you wish you can add yours to the list separated by comma and after mine

- It is fine for me to add authors to the list but the only thing I ask is to do it after my name separated by commas, 
  but my plan was to add names myself of anyone giving major contribution spending time on it like you.

do not take it personal this are the future contributions rules to merge to the master branch which I will add to the instructions once ill have a bit of time